app_dart/lib/src/service/agent_service.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:math';
+
+import 'package:dbcrypt/dbcrypt.dart';
+import 'package:meta/meta.dart';
+
+/// Service class for Agent.
+///
+/// This service exists to provide common agent queries made by
```

Comment:
Nit: "queries" isn't quite right.  Maybe "This service provides functionality for interacting with [Agent] instances."

Reply:
Updated.